src/test/java/com/github/checkstyle/regression/extract/ExtractInfoProcessorTest.java

```diff
@@ -0,0 +1,66 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2017 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.github.checkstyle.regression.extract;
+
+import static com.github.checkstyle.regression.internal.TestUtils.assertUtilsClassHasPrivateConstructor;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.spy;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+
+import org.junit.Test;
+
+import com.github.checkstyle.regression.data.ModuleExtractInfo;
+
+public class ExtractInfoProcessorTest {
+    @Test
+    public void testIsProperUtilsClass() throws Exception {
+        assertUtilsClassHasPrivateConstructor(ExtractInfoProcessor.class);
+    }
+
+    @Test
+    public void testGetNameToModuleInfoFromInputStream() {
+        final InputStream is = ExtractInfoProcessor.class.getClassLoader()
+                .getResourceAsStream("checkstyle_modules.json");
+        final Map<String, ModuleExtractInfo> map =
+                ExtractInfoProcessor.getNameToModuleExtractInfoFromInputStream(is);
+        assertEquals(165, map.size());
```

Comment:
OK. And, IMO, it is better to create another json file only for `ExtractInfoProcessorTest`, with a name like `checkstyle_modules_sample.json`(and maybe in `resources/.../extract`).